### PR TITLE
Legend updates

### PR DESCRIPTION
**Pull request by Will-lamuth**
- removed the old php files that weren't being used. 
- updated legend HTML class names in index.html to better match the current legend
- changed Chesapeake overlay color to blue #5a5fcf
- Changed popup scroll so the Hydrocode header is always visible
- Added states to the legend with their own distinct colored icons
- VA marker updated with use types and have their own distinct color

### CODE REVIEW

File: static/js/legend.js [hunk 1]
```diff
@@ -31,8 +31,9 @@ export function setMapOnAll(map, Tmarkers, id=null) {
     }
   
     if(map!=null){
-      markerCluster.addMarkers(Tmarkers);
-      console.log('markeres added');
+        markerCluster.removeMarkers(Tmarkers);
+        markerCluster.addMarkers(Tmarkers);
+        console.log('markeres added');
```
Comment (Pham-Vincent) [inline static/js/legend.js:36]: Remove This Console.log

File: static/js/legend.js [hunk 2]
```diff
@@ -70,13 +71,8 @@ export function selectAll(id, source){
...
     console.log(id);
 
-    //finds all checkboxes with name attritbute = box
-    if(id === "Select All UseTypes"){
-        checkboxes = document.getElementsByName("state");
-    }
-    else{
-        checkboxes = document.getElementsByName("box");
-    }
+    //finds checkboxes with name "box"
```
Comment (Pham-Vincent) [inline static/js/legend.js:74]: What Does This Even Mean?

File: static/js/script.js [hunk 3]
```diff
@@ -35,24 +35,25 @@ async function initMap() {
...
+    zoom: 8.2,
   
     //Mappitng Styles:
     //366d3e13ce470bd7 -> No Background Signs/Feature Styling Disabled
     //45c77a2db5a260c8 -> Background Signs/Feature Styling Enabled
     mapId: "366d3e13ce470bd7", 
     scrollwheel:true, //bypasses command+scroll to zoom
     streetViewControl: false, //removes streetview pegman
-    fullscreenControl: false,
+    fullscreenControl: false,
```
Comment (Pham-Vincent) [inline static/js/script.js:47]: Write Comment on what These Do

File: static/js/script.js [hunk 4]
```diff
@@ -35,24 +35,25 @@ async function initMap() {
...
   
     //Mappitng Styles:
     //366d3e13ce470bd7 -> No Background Signs/Feature Styling Disabled
     //45c77a2db5a260c8 -> Background Signs/Feature Styling Enabled
     mapId: "366d3e13ce470bd7", 
     scrollwheel:true, //bypasses command+scroll to zoom
     streetViewControl: false, //removes streetview pegman
-    fullscreenControl: false,
+    fullscreenControl: false, 
+    mapTypeControl: false,
```
Comment (Pham-Vincent) [inline static/js/script.js:48]: Write Comment on What These Do

File: static/js/script.js [hunk 5]
```diff
@@ -104,8 +105,8 @@ $.ajax({
 
         // Attach custom properties to the marker object
         marker.descriptions = {
-          description1: desc1,
-          description2: locality,
+          description2: desc1,
```
Comment (Pham-Vincent) [inline static/js/script.js:108]: Comment On What Each Of These Are?

File: templates/index.html [hunk 6]
```diff
@@ -162,18 +166,19 @@ <h3>Legend</h3>
             const arrow = document.querySelector('.arrow');
             arrow.classList.toggle('rotate180');
         }
-        function toggleDropDownMD(event){
+        function toggleDropDownUseType(event){
```
Comment (Pham-Vincent) [inline templates/index.html:169]: Comment The USe of this Function

File: templates/index.html [hunk 7]
```diff
@@ -162,18 +166,19 @@ <h3>Legend</h3>
...
+          const dropdown = document.querySelector('.drop-down-UseTypes');
           dropdown.classList.toggle('hidden');
-          const arrow = document.querySelector('.drop-down-arrow-MD');
-          arrow.classList.toggle('rotate-drop-down-180-MD');
+          const arrow = document.querySelector('.drop-down-arrow-UseType');
+          arrow.classList.toggle('rotate-arrow-UseType');
         }
-        function toggleDropDownVA(){
-          const dropdown = document.querySelector('.drop-down-VA');
+        function toggleDropDownState(event){
```
Comment (Pham-Vincent) [inline templates/index.html:176]: Comment What This Function Does

**Review by Pham-Vincent — CHANGES_REQUESTED** [review]
Code Looks Overall Good Just need to add some comments on some things And Make Some Comments more Clear And Also Delete PyCache

File: static/js/legend.js [hunk 8]
```diff
@@ -11,14 +11,12 @@ Date: 04/25/24
...
 
 //sets all markers in given array to visible or invisible(used for legend)
 export function setMapOnAll(map, Tmarkers, id=null) {
     //this removes Virginia points, as id == null and removes clustering on all Virginia
     if(map==null){
         markerCluster.removeMarkers(Tmarkers);
-        console.log('markeres removed');
     }
   
     for (let i = 0; i < Tmarkers.length; i++) {
```
Comment (Pham-Vincent) [inline static/js/legend.js:22]: Add a Comment Cuh

File: static/js/script.js [hunk 9]
```diff
@@ -89,7 +90,7 @@ $.ajax({
 
         //custom marker
         const glyphElement = new PinElement({
-          background: 'orange',
+          background: '#e06666',
```
Comment (Pham-Vincent) [inline static/js/script.js:93]: What Color is This?

File: static/js/script.js [hunk 10]
```diff
@@ -189,7 +190,7 @@ $.ajax({
 
       //custom marker
       const glyphElement = new PinElement({
-        background: 'orange',
+        background: '#fe9f3b',
```
Comment (Pham-Vincent) [inline static/js/script.js:193]: What Color?

**Review by Pham-Vincent — CHANGES_REQUESTED** [review]
Overall Good But Delete PyCache

**Review by snowcov — APPROVED** [review]
Everything new looks good, Nice Comments!

**Pull request merged**